src/lib/sampled.ts

```diff
@@ -127,19 +126,15 @@ export async function _sampledUrls(sitemapXml: string): Promise<string[]> {
       projDir += '/';
     }
 
-    const dirTreeRes = dirTree(projDir + 'src/routes');
-    routes = extractPaths(dirTreeRes);
-    // Match +page.svelte or +page@.svelte (used to break out of a layout).
-    //https://kit.svelte.dev/docs/advanced-routing#advanced-layouts-breaking-out-of-layouts
-    routes = routes.filter((route) => route.match(/\+page.*\.svelte$/));
+    routes = Object.keys(import.meta.glob('../../src/routes/**/+page*.svelte'));
```

Comment:
Did you try using sampled paths method after this change?

iirc, the reason I had to use a function, instead of `import.meta.glob` here, was due to `import.meta.glob` causing an error since it's evaluated at build time.

Reply:
I only changed this because I was having issues with `const dirTreeRes` being null on my Windows machine when I ran the tests or used the method on my Windows laptop:

```
An error occurred: TypeError: Cannot read properties of null (reading 'path')
    at extractPaths (C:/Users/jade/Code/super-sitemap/dist/sampled.js:270:13)
    at _sampledUrls (C:/Users/jade/Code/super-sitemap/dist/sampled.js:130:18)
    at Module.sampledUrls (C:/Users/jade/Code/super-sitemap/dist/sampled.js:41:18)
```

I've tested the method in a different repo now and it errors out. I reverted that change in https://github.com/JadedBlueEyes/super-sitemap/tree/jade/lang-param-dirtree, but it still has the original issue on both my machines, Windows and Linux. 

Possibly something to make a separate issue.

Reply:
oh ok. Either way. If you leave `import.meta.glob`, I'll try `sampledPaths()` in a project now to see if it works. But it's okay to leave it and submit another issue if prefer. (It's not something I've encountered on MacOS.)

Reply:
@JadedBlueEyes Remember to re-add the file containing the method definition, if reverting it, please.